src/tests/EdFi.DataManagementService.Tests.E2E/Features/Security/TokenIntrospection.feature.disabled

```diff
@@ -16,42 +16,51 @@ Feature: JWT Token Introspection
             Given the Data Management Service must receive a token issued by "http://localhost"
 
     Rule: Discovery API does not require an authorization token
+        @ignore
```

Comment:
The `@ignore ` here will help us filtering the existing tests that are marked as ignore, even if the file is disabled is hard to track tests that are not being executed.